packages/transaction-controller/src/types.ts

```diff
@@ -689,3 +741,24 @@ export type DefaultGasEstimates = {
    */
   maxPriorityFeePerGas?: string;
 };
+
+export type SwapOptions = {
```

Comment:
Do we need a type for this? It's not used anywhere else and we don't use types for any of the other arguments to `addTransaction`?

Reply:
Removed in https://github.com/MetaMask/core/pull/1877/commits/54ec5d3b7a2cef7e859ef1335c34fd8f64cd2e3d